docs.json

```diff
@@ -1,66 +1,34 @@
 {
-  "name": "React Query Firebase",
+  "name": "TanStack Query Firebase",
   "twitter": "invertaseio",
   "theme": "#FF4154",
   "docsearch": {
     "apiKey": "8838b79166d4839463db6aeafbbaa383",
-    "indexName": "react-query-firebase"
+    "indexName": "tanstack-query-firebase"
```

Comment:
Can't change this one yet, as it is not valid on the docsearch side.

Reply:
oh i see, let me make a revert here